python/ray/dashboard/dashboard.py

```diff
@@ -171,7 +174,7 @@ def get_forbidden(_) -> aiohttp.web.Response:
 
     def log_dashboard_url(self):
         url = "http://{}:{}?token={}".format(self.ip, self.port, self.token)
-        with open("/tmp/ray/dashboard_url", "w") as f:
+        with open(os.path.join(self.temp_dir, "dashboard_url"), "w") as f:
```

Comment:
`/tmp/ray` is shared by all Ray sessions, but the dashboard is specific to a single Ray session I believe, so I moved this file into `/tmp/ray/session...`.